From 9a038b0d9aede089dc8ab1705e1ea29dd231bd46 Mon Sep 17 00:00:00 2001
From: Stephen <stephen@vamrs.com>
Date: Fri, 13 May 2022 20:27:56 +0800
Subject: [PATCH 18/21] input: touchscreen: focaltech touch: Fix compilation
 error

It fixes the issue with the following logs.

  /opt/gcc-arm-10.3-2021.07-x86_64-aarch64-none-linux-gnu/bin/aarch64-none-linux-gnu-ld: drivers/built-in.a: member drivers/input/touchscreen/focaltech_touch/focaltech_test in archive is not an object
  Makefile:1284: recipe for target 'vmlinux' failed
  make: *** [vmlinux] Error 1
   MAKE KERNEL IMAGE FAILED.

Signed-off-by: Stephen <stephen@vamrs.com>
Signed-off-by: Yuntian Zhang <yt@radxa.com>
---
 drivers/input/touchscreen/focaltech_touch/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/focaltech_touch/Makefile b/drivers/input/touchscreen/focaltech_touch/Makefile
index fbf30651f2f8..9d022eb01a84 100644
--- a/drivers/input/touchscreen/focaltech_touch/Makefile
+++ b/drivers/input/touchscreen/focaltech_touch/Makefile
@@ -15,5 +15,5 @@ focaltech-ts-y += focaltech_esdcheck.o
 focaltech-ts-y += focaltech_i2c.o
 focaltech-ts-y += focaltech_point_report_check.o
 focaltech-ts-y += focaltech_upgrade_ft8201.o
-focaltech-ts-y += focaltech_test/
+#focaltech-ts-y += focaltech_test/
 
-- 
2.36.1

